src/components/savedAccounts/savedAccounts.js

```diff
@@ -57,6 +57,18 @@ class SavedAccounts extends React.Component {
     e.stopPropagation();
   }
 
+  handleRemovePassphrase(account, e) {
+    e.stopPropagation();
+
+    const { savedAccounts } = this.props;
+    const savedActiveAccount = savedAccounts.find(acc => `${acc.network} ${acc.network}` === `${account.network} ${account.network}`);
```

Comment:
This doesn't look right. What if there are two accounts on the same network?

There should be some tests with multiple unlocked accounts that catch it.